src/app/dashboard/roles/page.tsx

```
@@ -162,14 +178,18 @@ export default function RolesPage() {
+      {filteredRoles.length === 0 && !loading && (
         <div className="text-center py-12">
-          <div className="text-gray-500 mb-4">No roles found</div>
+          <div className="text-gray-500 mb-4">
+            {statusFilter === 'active' && 'No active roles found'}
+            {statusFilter === 'inactive' && 'No inactive roles found'}
+            {statusFilter === 'all' && 'No roles found'}
+          </div>
           <button
             onClick={() => handleOpenModal()}
             className="bg-amber-600 text-white px-4 py-2 rounded-lg hover:bg-amber-700"
           >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)